app/src/main/java/org/fossasia/susi/ai/helper/Utils.kt

```diff
@@ -18,8 +19,10 @@ object Utils {
     }
 
     fun getImageLink(skillData: SkillData): String {
-        val uri = URI("${BaseUrl.SUSI_DEFAULT_BASE_URL}/cms/getImage.png?model=${skillData.model}&language=${skillData.language}&group=${skillData.group}&image=${skillData.image}")
-        return uri.toASCIIString()
+        val s = "${BaseUrl.SUSI_DEFAULT_BASE_URL}/cms/getImage.png?model=${skillData.model}&language=${skillData.language}&group=${skillData.group}&image=${skillData.image}"
+                .replace(" ","%20")
```

Comment:
Obviously, you are not going to release new app right away. Add tests now as previously instructed. In order to save time in that PR, you are going to spend it here 

Also, I told you that save the single letter variable names for your college projects